### ISSUE TITLE
Consider using lbl:latitude and lbl:longitude to display results

### ISSUE THREAD

**Issue by stepps00** [whosonfirst/whosonfirst-www-api#120]
https://places.mapzen.com/search?q=city+by+the+bay&scope=alt

The current use of `geom:latitude` and `geom:longitude` is causing the above search to display San Francisco in the Pacific. Defaulting to the `lbl:latitude` and `lbl:longitude` values would solve this issue and similar cases, as we curate the `lbl:` properties for map display.

**Comment by thisisaaronland** [comment]
Yes, that's a bug. We should probably document the logic here as a reference for all the software-bits to implement:

https://github.com/whosonfirst/go-whosonfirst-geojson-v2/blob/master/properties/whosonfirst/whosonfirst.go#L65-L92
